### DISCUSSION TITLE
Make the team score difference display less invasive

### DISCUSSION THREAD

**Discussion by felipemarins** [ppy/osu#28992] [Ideas]
The lack of an option to hide the leaderboard on multiplayer has been a somewhat frequent complaint by players at Lazer Grand Arena. And since the developers have already said that it's unlikely to add the option to hide it, I think that decreasing the team score difference display could make it at least better, since it sometimes can go over the playfield. So I guess decreasing its width may be a good option.

**Comment by peppy** [comment]
I've also brought this up internally. Can you be more specific about the parts which bring you distress? Is it the size of the display? The animations? The opacity?

Including annotated screenshots may help in explaining things.

**Reply by felipemarins** [reply]
I'm sorry, I think I'm not the right person to answer it, as I don't play multi. I just opened this discussion because I thought that it would be a good option to reduce the complaints about it being distracting. I can only give my opinion as an spectator. If you're fine with that, my only complaint would be that it sometimes go over the circles, specially if the blue team is winning by a large margin, which makes the HUD feel less organized and a bit cramped (I don't know if that would be the right term, but it feels like it uses a space that it should not be using). But, again, I'm sorry for not being able to give a player's perspective on this.
